contracts/registry/README.md

```diff
@@ -1 +1,14 @@
+# SBT opt-in
```

Comment:
First level header is usually a title, and there should be only one first level header.
Below the header let's start with a summary, and link to the registry documentation of NEP-393.

Reply:
DONE